Misc/NEWS.d/next/Library/2018-06-06-22-01-33.bpo-33274.teYqv8.rst

```diff
@@ -0,0 +1,3 @@
+W3C DOM Level 1 specifies return value of Element.removeAttributeNode() as
+"The Attr node that was removed." xml.dom.minidom now complies with this
+requirement
```

Comment:
Trailing period, please.  (Yes, I'm picky!)

Reply:
Like I said on the issue page, all I'm doing is bringing the code into compliance with the docs. The only thing I can think of to put in the doc is edit *previous* versions to say this one function does something different from what the spec says.

I have made the requested changes; please review again